WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetailsViewController+LocalizedStrings.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+
+extension RegisterDomainDetailsViewController {
+    enum Localized {
+        enum PrivacySection {
+            static let title = NSLocalizedString(
+                "Privacy Protection",
+                comment: "Register Domain - Privacy Protection section header title"
+            )
+            static let description = NSLocalizedString(
+                "Domain owners have to share contact information in a public database of all domains. With Privacy Protection, we publish our own information instead of yours and privately forward any communication to you",
```

Comment:
I know it's not in the design, but could you please add a period at the end of this sentence? Thank you!

Reply:
Sure